modules/ROOT/nav.adoc

```diff
@@ -9,7 +10,14 @@
 * xref:getting-started:start-here.adoc[Start Here!]
 * xref:getting-started:do-a-quick-install.adoc[Do a Quick Install]
 * xref:getting-started:look-at-the-results.adoc[Explore the Server Configuration]
-* xref:getting-started:try-a-query.adoc[Run Your First N1QL Query]
+* xref:tutorials:couchbase-tutorial-student-records.adoc[]
+  ** xref:tutorials:install-couchbase-server.adoc[]
+  ** xref:tutorials:buckets-scopes-and-collections.adoc[]
+  ** xref:tutorials:java-tutorial/install-couchbase-java-sdk.adoc[]
+  ** xref:tutorials:java-tutorial/creating-the-students-collection.adoc[]
+  ** xref:tutorials:java-tutorial/creating-the-courses-collection.adoc[]
+  ** xref:tutorials:java-tutorial/retrieving-documents.adoc[]
+  ** xref:tutorials:java-tutorial/adding-course-enrollments.adoc[]
```

Comment:
Reinstate `try-a-query.adoc` for now, at least until we've merged the developer how-to guides. 

```suggestion
* xref:getting-started:try-a-query.adoc[Run Your First N1QL Query]
```